src/pyshark/capture/pipe_ring_capture.py

```diff
@@ -38,10 +38,10 @@ def __init__(self, pipe, ring_file_size=1024, num_ring_files=2, ring_file_name=N
         :param override_prefs: A dictionary of tshark preferences to override, {PREFERENCE_NAME: PREFERENCE_VALUE, ...}.
         :param disable_protocol: Tells tshark to remove a dissector for a specifc protocol.
         """
-        if display_filter is not None:
+        if "display_filter" in kwargs.keys():
```

Comment:
You can remove this as well as the exception. I don't think it's needed. Again, just write in the doc that display filters are not supported.

If you feel strongly about having this exception, then it's better the previous way. This is bad because it "swallows" any kwargs the user may have inputted (say they misspelt use_json, it'll suddenly not fail but not work either)

Reply:
While I agree, I should pass them up to the super, I think it's important to do a hard exception for this since every other capture allows it.